source/server/config/http/buffer.cc

```diff
@@ -29,9 +31,9 @@ HttpFilterFactoryCb BufferFilterConfig::createFilterFactory(const Json::Object&
 }
 
 /**
- * Static registration for the buffer filter. @see RegisterNamedHttpFilterConfigFactory.
+ * Static registration for the buffer filter. @see RegisterFactory.
  */
-static RegisterNamedHttpFilterConfigFactory<BufferFilterConfig> register_;
+static Registry::RegisterFactory<BufferFilterConfig, NamedHttpFilterConfigFactory> register_;
```

Comment:
Having just worked on this code, IIRC it's now possible to remove "server/config/network/http_connection_manager.h" from includes and bazel deps. Similarly for network filters IIRC you only need registry.h and filter_config.h and can remove configuration_impl.h/bazel dep. (I actually thought about doing something like this but got lazy, so this is awesome, thanks).